cmd/usage.js

```diff
@@ -94,29 +94,30 @@ module.exports = ({ fork, length, key }) => {
   `
 
   const run = ansi.bold(cmd + ' run')
-  const runArgs = ansi.bold('<key|dir|alias> [...args]')
-  const runBrief = 'Run an application from a key or dir.'
+  const runArgs = ansi.bold('[flags] <key|dir|alias> [...app-args]')
+  const runBrief = 'Run an application from a key or dir. app-args are passed directly to the app.'
   const runExplain = `${run} ${runArgs}
 
     ${runBrief}
 
     ${ansi.bold('key')}    pear://<key>
     ${ansi.bold('dir')}    file://<absolute-path> | <absolute-path> | <relative-path>
     ${ansi.bold('alias')}  pear://<alias>
-
-    --dev                      Run the app in dev mode
-    --link=url                 Simulate deep-link click open
-    --store|-s=path            Set the Application Storage path
-    --tmp-store|-t             Automatic new tmp folder as store path
-    --checkout=n               Run a checkout, n is version length
-    --checkout=release         Run checkout from marked released length
-    --checkout=staged          Run checkout from latest version length
+    ${ansi.bold('flags')}  --dev                      Run the app in dev mode
+           --link=url                 Simulate deep-link click open
+           --store|-s=path            Set the Application Storage path
+           --tmp-store|-t             Automatic new tmp folder as store path
+           --checkout=n               Run a checkout, n is version length
+           --checkout=release         Run checkout from marked released length
+           --checkout=staged          Run checkout from latest version length
     ${ansi.dim(ansi.italic(`
-     pear run pear://u6c6it1hhb5serppr3tghdm96j1gprtesygejzhmhnk5xsse8kmy
-     pear run pear://keet
-     pear run file://path/to/an-app-folder
-     pear run path/to/an-app-folder --some args
-     `))}`
+    pear run pear://u6c6it1hhb5serppr3tghdm96j1gprtesygejzhmhnk5xsse8kmy
```

Comment:
leave positioning as is

Reply:
Is it on purpose that it's not aligned with the rest?

Reply:
it should be aligned (and currently is)

<img width="1003" alt="Screenshot 2024-02-22 at 17 41 03" src="https://github.com/holepunchto/pear/assets/1190716/3793ae4e-bed3-4f2f-9109-63817a097f79">